src/rust/src/metrics_ffi.rs

```diff
@@ -16,18 +16,10 @@ pub extern "C" fn metrics_run(
 ) -> bool {
     // Parse any allowed IPs.
```

Comment:
We're not parsing here anymore:
```suggestion
    // Convert the CString IPs to Rust strings.
```